use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -1,46 +1,222 @@
+
+### Add a tiering policy
+
+To add a tiering policy, connect to your $SERVICE_SHORT and call `add_tiering_policy`:
+
+```sql
+SELECT add_tiering_policy(hypertable REGCLASS, move_after INTERVAL, if_not_exists BOOL = false);
+```
+
+For example, tier chunks that are more than three days old in the `example` hypertable in the following way: 
+
+```sql
+SELECT add_tiering_policy('example', INTERVAL '3 days');
+```
+
+### Remove a tiering policy
+
+To remove an existing tiering policy, call `remove_tiering_policy`:
+
+```sql
+SELECT remove_tiering_policy(hypertable REGCLASS, if_exists BOOL = false);
+```
+
+For example, remove the tiering policy from the `example` hypertable in the following way:
+
+```sql
+SELECT remove_tiering_policy('example');
+```
+
+If you remove a tiering policy, new scheduled chunks are not be tiered. However, chunks in tiered storage are not untiered. You [untier chunks manually](#manually-tier-and-untier-chunks) to local storage.
+
+## Manually tier and untier chunks
+
+If tiering policies do not meet your current needs, you can tier and untier chunks manually. 
+
+### Tier chunks
+
+Tiering a chunk is an asynchronous process that schedules the chunk to be tiered. In the following example, you tier chunks older than three days in the example [hypertable][hypertable]. You then list the tiered chunks.
+
+<Procedure>
+
+1. At the psql prompt, select all chunks in the table `example` that are older
+   than three days:
+
+   ```sql
+   SELECT show_chunks('example', older_than => INTERVAL '3 days');
+   ```
+
+    This returns a list of chunks. Take a note of the chunk names:
+
+   ```sql
+   |1|_timescaledb_internal_hyper_1_2_chunk|
+   |2|_timescaledb_internal_hyper_1_3_chunk|
+   ```
+
+1. Call the `tier_chunk` function to manually tier each chunk:
+
+   ```sql
+   SELECT tier_chunk( '_timescaledb_internal_hyper_1_2_chunk');
+   ```
+
+1. Repeat for all chunks you want to tier.
+
+1. To see which chunks are tiered into the object storage tier, use the `tiered_chunks` informational view:
+
+    ```sql
+    SELECT * FROM timescaledb_osm.tiered_chunks;
+    ```
+
+</Procedure>
+
+Tiering a chunk schedules it for migration to the object storage tier, but the migration won't happen immediately. Chunks are tiered one at a time in order to minimize database resource consumption. You can continue to query a chunk during migration.
+
+To see which chunks are scheduled for tiering either by policy or by a manual call, but have not yet been tiered, use this view:
+
+```sql
+SELECT * FROM timescaledb_osm.chunks_queued_for_tiering ;
+```
+
+### Untier chunks
+
+Tiered data is immutable. To update data in a tiered chunk, move it back to local storage, that is, Timescale's standard high-performance storage tier. You can do so by using the `untier_chunk` stored procedure.
+
+Untiering chunks is a synchronous process. Chunks are renamed when the data is untiered.
```

Comment:
Don't see this should be on a newline. 

Can you add a lead in sentence before the procedure please.